scripts/apps/vocabularies/views/vocabulary-config.html

```diff
@@ -34,24 +34,38 @@
     </div>
 
     <div class="sd-page__content">
-      <div ng-if="loading" class="loading-overlay" ng-class="{active: loading}" style="opacity: 0.5;"></div>
-        <ul class="pills-list">
-          <li ng-repeat="vocabulary in vocabularies | orderBy:'display_name|translate' | filter: query track by vocabulary._id" ng-if="matchFieldTypeToTab(tab, vocabulary.field_type)">
-            <h6>{{ vocabulary.display_name | translate }}</h6>&nbsp;
-            <em><strong ng-show="!vocabulary.field_type">({{ vocabulary.items.length }})</strong></em>
-            <div class="actions">
-              <button
-                ng-click="openVocabulary(vocabulary)"
-                title="{{:: 'Edit vocabulary' | translate }}"
-              ><i class="icon-pencil"></i></button>
-              <button
-                ng-click="remove(vocabulary)"
-                ng-if="tab !== 'vocabularies'"
-                title="{{:: 'Remove vocabulary' | translate }}"
-              ><i class="icon-trash"></i></button>
+        <div ng-if="loading" class="loading-overlay" ng-class="{active: loading}" style="opacity: 0.5;"></div>
+
+        <div ng-repeat="tag in tags" ng-if="existsVocabulariesForTag(tag, tab)">
+            <div className='sd-list-header'>
+                <span className="sd-list-header__name">{{tag.text}}</span>
```

Comment:
Label styles don't look [right](https://user-images.githubusercontent.com/2076953/49014107-a53e4280-f187-11e8-9e29-29d7d8d65727.png)

Reply:
I used the style required by @fritzSF.
Anyway @fritzSF  will take a look and will do some UI improvements on this list.